ci-operator/config/openshift-psap/ci-artifacts/openshift-psap-ci-artifacts-master__4.6.yaml

```diff
@@ -27,11 +21,16 @@ resources:
       memory: 2Gi
 tests:
 - as: gpu-operator-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
```

Comment:
This sounds a little bit too strict, 

![image](https://user-images.githubusercontent.com/10882062/127120856-96cb923f-e24d-4c10-92c1-4ef1669f6941.png)

Reply:
hum indeed, I've changed it to 1h30
```
INFO[2021-07-22T22:10:45Z] Running step gpu-operator-e2e-presubmit-operatorhub. 
INFO[2021-07-22T22:38:55Z] Step gpu-operator-e2e-presubmit-operatorhub succeeded after 28m10s. 
INFO[2021-07-22T22:38:55Z] Running step gpu-operator-e2e-presubmit-operatorhub-cleanup. 
INFO[2021-07-22T22:39:45Z] Step gpu-operator-e2e-presubmit-operatorhub-cleanup succeeded after 50s. 
INFO[2021-07-22T22:39:45Z] Running step gpu-operator-e2e-presubmit-master. 
INFO[2021-07-22T23:04:26Z] Step gpu-operator-e2e-presubmit-master succeeded after 24m40s. 
```


https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift-psap_ci-artifacts/214/pull-ci-openshift-psap-ci-artifacts-master-gpu-operator-e2e/1418320896631771136